web/packages/shared/components/UnifiedResources/UnifiedResources.story.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { ButtonBorder, Flex } from 'design';
+
+import { apps } from 'teleport/Apps/fixtures';
+import { databases } from 'teleport/Databases/fixtures';
+import { kubes } from 'teleport/Kubes/fixtures';
+import { desktops } from 'teleport/Desktops/fixtures';
+import { nodes } from 'teleport/Nodes/fixtures';
+
+import { UrlResourcesParams } from 'teleport/config';
+import { ResourcesResponse } from 'teleport/services/agents';
+
+import { UnifiedResources } from './UnifiedResources';
+import { SharedUnifiedResource } from './types';
+
+export default {
+  title: 'Shared/UnifiedResources',
+};
+
+const aLotOfLabels = {
+  ...databases[0],
+  name: 'A DB with a lot of labels',
+  labels: Array(300)
+    .fill(0)
+    .map((_, i) => ({ name: `label-${i}`, value: `value ${i}` })),
+};
+
+const allResources = [
+  ...apps,
+  aLotOfLabels,
+  ...databases,
+  ...kubes,
+  ...desktops,
+  ...nodes,
+  ...apps,
+  ...databases,
+  ...kubes,
+  ...desktops,
+  ...nodes,
```

Comment:
why duplicate spreads?

Reply:
I think it was done to show more cards in the story (I didn't change it).

Reply:
Ok sounds fine to me